### PR TITLE
feat: tadded todo-list and todo-item components

### PR DESCRIPTION
**Pull request by Karo129**

### CODE REVIEW

File: components/todo-list/todo-list.component.ts [hunk 1]
```diff
@@ -7,9 +9,45 @@ import { Component, OnInit } from '@angular/core';
 })
 export class TodoListComponent implements OnInit {
 
-  constructor() { }
+  newTodo: any;
```
Comment (thesobek) [inline components/todo-list/todo-list.component.ts]: I would define model for this. However at this moment `any` is fine.

File: components/todo-list/todo-list.component.html [hunk 2]
```diff
@@ -1 +1,15 @@
-<p>todo-list works!</p>
+<main class="todo">
+    <div class="section-main">
+        <!-- <app-header></app-header> -->
+        <header class="header">
+            <h1>todos</h1>
+            <i class="fa-solid fa-chevron-down toggleButton" (click)="toggleAll()"></i>
+            <input type="text" class="new-todo" placeholder="What needs to be done?" [(ngModel)]="newTodo"
```
Comment (thesobek) [inline components/todo-list/todo-list.component.html:7]: This is ok. But lets talk about using here 'reactive forms'

**Comment by thesobek** [comment]
Looks good to me 🏆

With minor comments we need to resolve/talk later

File: components/todo-list/todo-list.component.html (hunk 2 above)
Comment (Karo129) [reply components/todo-list/todo-list.component.html:7]: ok, I'll leave it for now.

**Pull request merged**